app/src/Event/EventController.php

```diff
@@ -20,8 +20,15 @@
 
 class EventController extends BaseController
 {
-    private $itemsPerPage = 10;
-    private $pendingItemsPerPage = 30;
+    private $itemsPerPage;
+    private $pendingItemsPerPage;
+
+    public function __construct(Slim $app)
```

Comment:
When we're using ```Slim``` here we should also change occurences for ```\Slim\Slim``` with ```Slim``` everywhere else in the file…